lib/components/primitive-components/Trace/Trace.ts

```diff
@@ -862,7 +861,7 @@ export class Trace
 
     const portIds = connectedPorts.map((p) => p.port.schematic_port_id).sort()
     const portPairKey = portIds.join(",")
-    const board = this.parent as Board
+    const board = this.root?._getBoard()
```

Comment:
this should be operating on the subcircuit, not the board, otherwise subcircuits aren't isolated (subcircuits have a major optimization strategy where they render independently from the board)